.github/workflows/migrations.yml

```diff
@@ -21,6 +21,6 @@ jobs:
         npm run migration-persist
         git diff --exit-code backup/**
     - name: check filesystem migrations have been added to database
-      run: npm run migration-list
+      run: npm run migration-sync
```

Comment:
don't forget to restore

Reply:
list is right here, if it fails, run migration-list manually and up the missing migrations before commit.

Reply:
So back change.